mkdocs.yml

```diff
@@ -125,123 +126,129 @@ plugins:
 
 # Navigation
 nav:
-  - Home: index.md
-  - Get started:
-    - get-started/index.md
-    - Python: get-started/python.md
-    - Java: get-started/java.md
-  - Build your agent:
-    - tutorials/index.md
-    - Multi-tool agent: get-started/quickstart.md
-    - Agent team: tutorials/agent-team.md
-    - Streaming agent:
-      - get-started/streaming/index.md
-      - Python: get-started/streaming/quickstart-streaming.md
-      - Java: get-started/streaming/quickstart-streaming-java.md
-    - Testing: get-started/testing.md
-    - Code samples: https://github.com/google/adk-samples
-    - Advanced setup: get-started/installation.md
-  - Technical overview: get-started/about.md
-  - Agents:
-    - agents/index.md
-    - LLM agents: agents/llm-agents.md
-    - Workflow agents:
-      - agents/workflow-agents/index.md
-      - Sequential agents: agents/workflow-agents/sequential-agents.md
-      - Loop agents: agents/workflow-agents/loop-agents.md
-      - Parallel agents: agents/workflow-agents/parallel-agents.md
-    - Custom agents: agents/custom-agents.md
-    - Multi-agent systems: agents/multi-agents.md
-    - Agent Config: agents/config.md
-    - Models & Authentication: agents/models.md
-  - Tools for Agents:
-    - tools/index.md
-    - Built-in tools: tools/built-in-tools.md
-    - Gemini API tools:
-      -  Computer use: tools/gemini-api/computer-use.md
-    - Google Cloud tools:
-      - Overview: tools/google-cloud-tools.md
-      - MCP Toolbox for Databases: tools/google-cloud/mcp-toolbox-for-databases.md
-      - Code Execution with Agent Engine: tools/google-cloud/code-exec-agent-engine.md
-    - Third-party tools:
-      - tools/third-party/index.md
-      - Exa: tools/third-party/exa.md
-      - Firecrawl: tools/third-party/firecrawl.md
-      - GitHub: tools/third-party/github.md
-      - Hugging Face: tools/third-party/hugging-face.md
-      - Agentic UI (AG-UI): tools/third-party/ag-ui.md
-  - Custom Tools:
-    - tools-custom/index.md
-    - Function tools:
-      - Overview: tools/function-tools.md
-      - Tool performance: tools/performance.md
-      - Action confirmations: tools/confirmation.md
-    - MCP tools: tools/mcp-tools.md
-    - OpenAPI tools: tools-custom/openapi-tools.md
-    - Authentication: tools/authentication.md
+  - Building Agents:
+    - Home:
+      - index.md
+    - Get started:
+      - get-started/index.md
+      - Python: get-started/python.md
+      - Java: get-started/java.md
+    - Build your agent:
+      - tutorials/index.md
+      - Multi-tool agent: get-started/quickstart.md
+      - Agent team: tutorials/agent-team.md
+      - Streaming agent:
+        - get-started/streaming/index.md
+        - Python: get-started/streaming/quickstart-streaming.md
+        - Java: get-started/streaming/quickstart-streaming-java.md
+      - Testing: get-started/testing.md
```

Comment:
"Testing" (rename: "Test" should go under "Running Agents", after the "Agent Runtime" topic

The "Testing" topic is a mess; it doesn't know what it wants to be, and it probably should be several pages, not one. Task: b/457218750